stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -71,6 +71,13 @@ extension NSString : ExpressibleByStringLiteral {
   }
 }
 
+extension NSString : _HasCustomAnyHashableRepresentation {
```

Comment:
I don't think it's good enough to add this to just NSString. Anything that has a bridgeable representation should be eagerly bridged, just to be safe.